### DISCUSSION TITLE
Dutch Language Update for 0-55-0

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#838] [General]
Added the 2 language strings for version 0-55-0. 

[hfs-lang-nl 0-55-0.zip](https://github.com/user-attachments/files/18215549/hfs-lang-nl.0-55-0.zip)

Kind regards,
Henk - https://fileserver.ph5hp.nl - https://ph5hp.nl

**Comment by rejetto** [comment]
thank you!
